runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java

```diff
@@ -51,10 +92,10 @@ public static void bootstrap(@Nonnull Layout layout,
      * @param retries                Number of retries to bootstrap each node before giving up.
      * @param retryTimeout           Duration between retries.
      */
-    public static void bootstrap(@Nonnull Layout layout,
-                                 @Nonnull CorfuRuntimeParameters corfuRuntimeParameters,
+    public static void bootstrap(@NonNull Layout layout,
```

Comment:
Could we be interested in bootstrapping only the management server and not the layout?

Reply:
Yes, we could. This is a generic utility to bootstrap the entire cluster. If you just want to bootstrap the management servers, we have the cmdlet utilities and ManagementCLient level APIs for the user.